src/IconButton/index.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { OverlayTrigger, Popover, PopoverTitle, PopoverContent } from '../';
+import InteractiveIcon from './InteractiveIcon';
+import useWindowSize from '../hooks/useWindowSize';
+
+const IconButton = ({
+  altText,
+  breakpoint,
+  darkBackground,
+  icon,
+  iconClassNames,
+  onClick,
+  popoverPlacementLg,
+  popoverPlacementSm,
+  popoverText,
+  popoverTitle,
+  popoverTrigger,
+}) => {
+  const windowSize = useWindowSize();
+  const placement = windowSize.width >= breakpoint ? popoverPlacementLg : popoverPlacementSm;
```

Comment:
Can we get this logic done with popper config on the popover? Ref: https://popper.js.org/docs/v2/modifiers/flip/#fallbackplacements

Reply:
oooo, yeah, that looks cool. Can I chat with you about how to use it in conjunction with Bootstrap, if that's something you've done before?